src/NuGet.Core/NuGet.Packaging/Signing/DerEncoding/DerSequenceReader.cs

```diff
@@ -560,8 +560,8 @@ private static void CheckTag(DerTag expected, byte[] data, int position)
 #if DEBUG
                     ,
                     new InvalidOperationException(
-                        "Expected tag '0x" + expectedByte.ToString("X2") +
-                            "', got '0x" + actual.ToString("X2") +
+                        "Expected tag '0x" + expectedByte.ToString("X2", CultureInfo.CurrentCulture) +
+                            "', got '0x" + actual.ToString("X2", CultureInfo.CurrentCulture) +
```

Comment:
Since it's binary data, we want the same representation in all languages.

```suggestion
                        "Expected tag '0x" + expectedByte.ToString("X2", CultureInfo.InvariantCulture) +
                            "', got '0x" + actual.ToString("X2", CultureInfo.InvariantCulture) +
```

Reply:
Dismissed since this is only affecting a user-facing error message, and both both byte data chunks will be shown with the same culture.